apps/anoma_client/lib/api/intents.ex

```diff
@@ -64,6 +64,19 @@ defmodule Anoma.Client.Api.Servers.Intents do
       "GRPC #{inspect(__ENV__.function)} request: #{inspect(request)}"
     )
 
-    %Verify.Response{valid: true}
+    res =
+      request.intent.intent
+      |> Nock.Cue.cue!()
+      |> Transaction.from_noun()
+      |> elem(1)
+      |> Anoma.RM.Intent.verify()
+
+    bool =
+      case res do
+        true -> true
+        _ -> false
+      end
```

Comment:
I feel we should have a general function that takes our error or true to a boolean in the module that has the weird signature on this. Maybe near the protocol module that will exist?

Reply:
hm yeah that would be fine

however do we generally have a lot of places where we need this? this is the first one i can recall

Reply:
One of the firsts, but it's a good helper to have